plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseCaseInsensitiveMappingTest.java

```diff
@@ -42,6 +46,13 @@
 
```

Comment:
did you consider making BaseCaseInsensitiveMappingTest be part of `Base(Jdbc)ConnectorTest`?
would it be horrible?

cc @losipiuk

Reply:
There are flakiness issues with that last I checked (with BigQuery).

The problem was that you can't isolate between tests. e.g. if you added a colliding schema name in a test all other unrelated tests will start failing because they don't have `case-insensitive-name-matching` enabled.

Reply:
@hashhar maybe we could have a specific schema for testing those clashing table names. 
Testing clashing schema names should not generate failures I think, as schema listing should always work, right?

Not sure if effort is worth gain, though.